cmd/alertmanager/main.go

```diff
@@ -216,8 +220,8 @@ func main() {
 		inhibitor.Stop()
 		disp.Stop()
 
-		inhibitor = NewInhibitor(alerts, conf.InhibitRules, marker)
-		disp = NewDispatcher(alerts, NewRoute(conf.Route, nil), build(conf.Receivers), marker)
+		inhibitor = inhibit.NewInhibitor(alerts, conf.InhibitRules, marker)
```

Comment:
I'm fine with either way, especially since it wouldn't work with the "Dispatcher" used above as there are multiple components to that package. And then mixing the two in consecutive lines of code would seem odd.